packages/atomic/cypress/e2e/search-box/text-area-search-box.cypress.ts

```diff
@@ -0,0 +1,633 @@
+import {
```

Comment:
You're right about all that. I'm just not 100% certain of all the search box features that might be affected by this change so this is being extra cautious.
I figured if i want this to work exactly like the original search box even after replacing a central piece then might as well run all the same tests to be sure.

Reply:
Okay, so next best thing, could we instead refactor the search box test suite so that it runs for both type of options  (textarea=true/false) ?

Something like: 

```typescript
describe('Search Box Test Suites', () => {
   ['true','false'].forEach(textAreaOption=> {

        function setupSearchbox() {
            return addSearchBox({
              suggestions: {
                maxWithoutQuery: maxSuggestionsWithoutQuery,
                maxWithQuery: numOfSuggestions,
              },
              recentQueries: {
                maxWithoutQuery: maxRecentQueriesWithoutQuery,
                maxWithQuery: numOfRecentQueries,
              },
              props: {
                'number-of-queries': numOfSuggestions + numOfRecentQueries,
                'suggestion-timeout': 2000,
                'text-area': textAreaOption
              },
            });
        }

         // [... all the tests]
   })
})
```

Even if it duplicates long running test (search box tests takes about ~7~8 minutes to run, multiplied by 2, multiplied by thousands of build in the next few months), at least they are written only once.

I would however strongly prefer if we took the time to reason about the actual differences, and only added the new specific tests that needs to be performed 🥺

Reply:
The facet tests take 45 minutes to complete so at least it wouldn't slow anyone down 😛 
But i hear you. I'll trim it down.